src/clarity-angular/popover/signpost/signpost.specs.ts

```diff
@@ -0,0 +1,485 @@
+            expect(signpostContent).toBeNull();
+
+            fixture.detectChanges();
+            // Test it shows after changes
+            signpostContent = clarityElement.querySelector(".signpost-content");
+            expect(signpostContent).not.toBeNull();
+            expect(signpost.ifOpenService.open).toBe(true);
+
+            // Hide it
+            signpostToggle.click();
+            fixture.detectChanges();
+            signpostContent = clarityElement.querySelector(".signpost-content");
+            // Test that its hidden
+            expect(signpostContent).toBeNull();
+            expect(signpost.ifOpenService.open).toBe(false);
+        });
+
+        it("takes an input for position", function() {
+            fixture.componentInstance.position = "top-middle";
+            fixture.detectChanges();
+            expect(signpost.signpostPosition).toBe("top-middle");
+        });
+
+        it("does not allow multiple open poovers", function() {
+            expect(signpost.signpostOptions.allowMultipleOpen).toBe(false);
+        });
+
+        it("has a default signpost content position", function() {
+            /*********
+             *
+             * @description
+             * There are five things to test here
+             * 1. Correct class name
+             * 2. correct anchor point
+             * 3. correct popover point
+             * 4. Correct Y offset
+             * 5. Correct X offset
+             *
+             */
+
+            let testClass = clarityElement.querySelector(".right-middle");
+            expect(testClass).toBeDefined();
+
+            // anchor point == Point.RIGHT_CENTER
+            expect(signpost.anchorPoint).toBe(Point.RIGHT_CENTER);
+            // PopoverPoint = Point.LEFT_CENTER
+            expect(signpost.popoverPoint).toBe(Point.LEFT_CENTER);
+            // offsetY = 6
+            expect(signpost.signpostOptions.offsetY).toBe(6);
+            // offsetX = 14
+            expect(signpost.signpostOptions.offsetX).toBe(14);
+        });
+
+        it("has a top-left signpost content position", function() {
+            /*********
+             *
+             * @description
+             * There are five things to test here
+             * 1. Correct class name
+             * 2. correct anchor point
+             * 3. correct popover point
+             * 4. Correct Y offset
+             * 5. Correct X offset
+             *
+             */
+
+            signpost.signpostPosition = "top-left";
+            fixture.detectChanges();
+
+            let testClass = clarityElement.querySelector(".top-left");
+            expect(testClass).toBeDefined();
+
+            expect(signpost.anchorPoint).toBe(Point.TOP_LEFT);
+            expect(signpost.popoverPoint).toBe(Point.BOTTOM_RIGHT);
+            expect(signpost.signpostOptions.offsetY).toBe(-8);
+            expect(signpost.signpostOptions.offsetX).toBe(10);
+        });
```

Comment:
Yeah, I got on the 3rd or 4th one and had the same thought but it conflicted with just getting the PR out for review. I'll revisit it if/when I can. 

Side note - I think I addressed all of the changes requested but I lost my comments in a page reload so let me know if anything is incorrect.